app-games/src/main/java/com/aptoide/android/aptoidegames/installer/UserActionDialog.kt

```diff
@@ -41,12 +49,38 @@ import com.aptoide.android.aptoidegames.design_system.PrimarySmallButton
 import com.aptoide.android.aptoidegames.theme.AGTypography
 import com.aptoide.android.aptoidegames.theme.AptoideTheme
 import com.aptoide.android.aptoidegames.theme.Palette
+import kotlinx.coroutines.launch
 
 @Composable
 fun UserActionDialog() {
   val viewModel = hiltViewModel<UserActionViewModel>()
   val state by viewModel.uiState.collectAsState()
   val context = LocalContext.current
+  val lifecycleOwner = LocalLifecycleOwner.current
+  var isOnForeground by remember { mutableStateOf(false) }
+  var installationActionLaunched by remember { mutableStateOf(false) }
+
+  LaunchedEffect(Unit) {
```

Comment:
Why not simply using the state as a key here instead of `snapshotFlow`? 🤔

Reply:
fixed